openpype/lib/file_transaction.py

```diff
@@ -212,3 +212,21 @@ def _same_paths(self, src, dst):
             return os.stat(src) == os.stat(dst)
 
         return src == dst
+
+
+def fill_cleanupFullPaths_for_representation(instance, repre):
```

Comment:
```suggestion
def add_expected_files_for_cleanup(instance, repre):
```

Reply:
If you would keep it, at least move it to `openpype.pipeline.publish` and rename it to key agnostic function name.

Reply:
Ok, moved.